Hangman/Hangman/main.swift

```diff
@@ -1,12 +1,211 @@
+
+var chances = 6
+
+let alph: Set<Character> = Set("abcdefghijklmnopqrstuvwxyz")
+
+var randomWord: [Character] = Array(words.randomElement() ?? "error")
+
+var hiddenWord: [Character] = Array(repeating: "_", count: randomWord.count)
+
+var wrongGuesses: [Character] = Array(repeating: "_", count: randomWord.count)
+
+//print(randomWord)
+
+let original = """
+
+⎡‾‾‾‾‾‾‾‾‾‾‾‾‾‾⎫
+⎜
+⎜
+⎜
+⎜
+⎜
+⎣________
+
+"""
+print(original)
+
+let guess1: String = """
+
+⎡‾‾‾‾‾‾‾‾‾‾‾‾‾‾⎫
+⎜             🥺
+⎜
+⎜
+⎜
+⎜
+⎣________
+
+
+"""
+
+let guess2: String = """
+
+⎡‾‾‾‾‾‾‾‾‾‾‾‾‾‾⎫
+⎜             🥺
+⎜              ⎜
+⎜              ⎜
+⎜
+⎜
+⎣________
+
+
+"""
+
+let guess3: String = """
+
+⎡‾‾‾‾‾‾‾‾‾‾‾‾‾‾⎫
+⎜             🥺
+⎜           ✌🏽 ⎜
+⎜              ⎜
+⎜
+⎜
+⎣________
+
+
+"""
+
+let guess4: String = """
+
+⎡‾‾‾‾‾‾‾‾‾‾‾‾‾‾⎫
+⎜             🥺
+⎜           ✌🏽 ⎜ ✋🏽
+⎜              ⎜
+⎜
+⎜
+⎣________
+
+
+"""
+
+let guess5: String = """
+
+⎡‾‾‾‾‾‾‾‾‾‾‾‾‾‾⎫
+⎜             🥺
+⎜           ✌🏽 ⎜ ✋🏽
+⎜              ⎜
+⎜            👞
+⎜
+⎣________
+
+
+"""
+
+let guess6: String = """
+
+⎡‾‾‾‾‾‾‾‾‾‾‾‾‾‾⎫
+⎜             😵
+⎜           👋🏽 ⎜👋🏽
+⎜              ⎜
+⎜            👞 👞
+⎜
+⎣________
+
+
+"""
+
+print("enter a character", hiddenWord)
+
+
+var indices = Set<Int>()
+
+
+while chances != 0  {
+
+let userInput = readLine() ?? "a"
+        
+for (index, char) in randomWord.enumerated() {
+    if userInput == String(char) {
+        indices.insert(index)
+        
+    }
+}
+
+for (index, _) in hiddenWord.enumerated() {
+    if indices.contains(index) {
+        hiddenWord[index] = Character(userInput)
+        indices.remove(index)
+//        indices.remove(at: index)
+//        print("here 1")
+        
+    } else {
+        for (index, char) in randomWord.enumerated() {
+            if userInput != String(char) {
+                wrongGuesses[index] = char
+//                print("here")
+            }
+        
+        }
+        
+    }
+    
+}
+
+if !randomWord.contains(Character(userInput)) {
+    print("Wrong letters \(userInput)")
+    chances -= 1
+    
+    switch chances {
+     case 0:
```

Comment:
This would have been a great place to also display what the hidden word had been!